x-pack/test/functional/apps/infra/feature_controls/infrastructure_security.ts

```diff
@@ -63,13 +63,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       it('shows infrastructure navlink', async () => {
         const navLinks = (await appsMenu.readLinks()).map((link) => link.text);
-        expect(navLinks).to.eql([
-          'Overview',
-          'Alerts',
-          'SLOs',
-          'Infrastructure',
-          'Stack Management',
-        ]);
+        expect(navLinks).to.eql(['Overview', 'Alerts', 'Infrastructure', 'Stack Management']);
```

Comment:
Perhaps it would be better if this test only tests for existence of the relevant navLinks for the context of this test instead of a bunch of navLinks its not responsible for?

Reply:
That sounds mega reasonable.